config/defaults_test.go

```diff
@@ -1,664 +1,664 @@
-/*
-Copyright 2020 gRPC authors.
+// /*
```

Comment:
Little confused about what is happening with this file, comment all of them out? If so, we would not have any unite test for defaults.go.

Reply:
Good catch!